features/cache_control_modification.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Proxy Modification of Cache-Control Header to include max-age when Babbage sets Cache-Control to "public" or "private"
+
+  Scenario Outline: Proxy adjusts Cache-Control header based on page release time context
+    Given Babbage will send the following response:
+      """
+      Mock response from Babbage
+      """
+    And Babbage will set the "Cache-Control" header to "<Babbage-Cache-Control>"
+    And the "/some-path" page <Release-Time-Context>
```

Comment:
This makes this test more complicated than it needs to be. You should remove this parameter, as it does not really affect this test.

Reply:
removed